extensions/typescript/src/features/referenceProvider.ts

```diff
@@ -14,7 +14,7 @@ export default class TypeScriptReferenceSupport implements ReferenceProvider {
 	public constructor(
 		private client: ITypescriptServiceClient) { }
 
-	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, token: CancellationToken): Promise<Location[]> {
+	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, _: (value: Location) => void, token: CancellationToken): Promise<Location[]> {
```

Comment:
@felixfbecker Thanks for the suggestion, that is much better! I actually don't need to provide a default parameter (we control the caller so this parameter is always passed). All I had to do was reorder the params.